HW5_2.py

```diff
@@ -0,0 +1,5 @@
+while True:
+    import time
```

Comment:
It is better to import packages at the very beginning of the file and don't import anything in loops